### PR TITLE
[test] Set a query-hybrid testcase as not critical

### PR DESCRIPTION
**Pull request by anyj0527**
Set the gstTestBackground TC as not critical:
  The another background pipeline (videotestsrc ! ... ! filesink) from
  `gstTestBackground` sometimes does not make test file properly.
  If following compare tests succeed, It means queryserver pipeline
  works well.

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

**Comment by anyj0527** [comment]
> plz update all script using gstTestBackground

I updated all occurrences, PTAL.

File: tests/nnstreamer_edge/edge/runTest.sh [hunk 1]
```diff
@@ -205,7 +205,7 @@ gstTestBackground "--gst-plugin-path=${PATH_TO_PLUGIN} \
         t. ! queue ! multifilesink location=raw6_%1d.log \
         t. ! queue ! edgesink port=0 connect-type=HYBRID dest-host=127.0.0.1 dest-port=${PORT} topic=tempTopic async=false" 6-1 0 0 30
 gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} \
-    edgesrc port=0 connect-type=HYBRID dest-host=127.0.0.1 dest-port=${PORT} topic=tempTopic num-buffers=10 ! multifilesink location=result6_0_%1d.log" 6-2 0 0 $PERFORMANCE
+    edgesrc port=0 connect-type=HYBRID dest-host=127.0.0.1 dest-port=${PORT} topic=tempTopic num-buffers=10 ! multifilesink location=result6_0_%1d.log" 6-2 1 0 $PERFORMANCE
```
Comment (jaeyun-jung) [inline tests/nnstreamer_edge/edge/runTest.sh:208]: not this line (line 206)

Comment (anyj0527) [reply tests/nnstreamer_edge/edge/runTest.sh]: Oops, I fixed it up. Thanks!

**Review by gichan-jang — APPROVED** [review]

**Review by niley7464 — APPROVED** [review]
LGTM!

**Pull request merged**